iscsi/iscsi.go

```diff
@@ -78,15 +78,17 @@ type Connector struct {
 	DoCHAPDiscovery bool `json:"do_chap_discovery"`
 }
 
+var version string = "1.0.0"
+
 func init() {
-	// by default we don't log anything, EnableDebugLogging() can turn on some tracing
-	debug = log.New(ioutil.Discard, "", 0)
+	// By default we produce info and error log entries which can be redirected using EnableDebugLogging()
+	klog.InfoS("[] csi-lib-iscsi", "version", version)
```

Comment:
Don't log during init: command line flags haven't been parsed yet, so this output will be emitted using the default klog settings.

It also will always produce output, whether the user of the library wants it or not.

Reply:
I do realize the flags have not been parsed yet, but since this init() is hidden inside the library, felt it was good to always log and logging a version produces a one time message of versioning info for the library. Can be removed without any loss if desired/required.

Reply:
Where did `klog.V(1).InfoS` not work as expected? If it was here, then not having command line flags explains it.

Reply:
Yes, please remove. A library should never make assumptions about what log output the user of the library wants.